pkg/test/application.go

```diff
@@ -8,6 +8,7 @@ import (
 func Application() kernel.Kernel {
 	options := []application.Option{
 		application.WithConfigFile("./config.dist.yml", "yml"),
+		application.WithKernelExitHandler(func(code int) {}),
```

Comment:
why do we have to overwrite this if there is a default with os.Exit?

Reply:
This one is used in tests where we can't use os.Exit..  
Furhtermore, this code will hopefully be gone soon..